src/orion/core/worker/consumer.py

```diff
@@ -197,9 +194,16 @@ def get_execution_environment(self, trial, results_file="results.log"):
 
         return env
 
-    def _consume(self, trial, workdirname):
+    def _prepare_config(self, trial, workdirname):
```

Comment:
Nit: I know your stance is to add type hints to new modules, but I think new methods might also be a good opportunity to add type hints.